src/app/pages/property-details/property-details.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, OnInit } from '@angular/core';
+import { PropertyDetailService } from '../../services/property-detail/property-detail.service';
+import { ActivatedRoute } from '@angular/router';
+import { NgxSpinnerService } from 'ngx-spinner';
+
+@Component({
+  selector: 'app-property-details',
+  templateUrl: './property-details.component.html',
+  styleUrls: ['./property-details.component.scss']
+})
+
+export class PropertyDetailsComponent implements OnInit {
+  property: Array<any> = [];
```

Comment:
@dorothyas  This holds a single property as returned by the service. Also, since you are searching with the slug(which is unique), there can only be one record. This should not be an `array`.

Reply:
@Victoradukwu  I have worked on all your feedback